### DISCUSSION TITLE
How hope is felt and what action(s) does hope inspire you to take?

### DISCUSSION THREAD

**Discussion by laurenwigmore** [life-itself/community#980] [General]
I wanted to open up a discussion around the idea of Hope. This is something I have personally been sitting with, and recently came to me in a meditation this morning.

**I wanted to gain people's perspectives on** 
**1. how hope is felt**
**2. what actions hope inspires or allows you to take**

I've purposefully left the word 'hope' undefined, as I'd love to gain insight into where this lands with people.

Drop your comments below or keep reading for previous reflections.

## Reflections and insights from the initial whatsapp group chat

1. For me I feel hope around me, and within me. It's a very visceral feeling of expansion and openness, with a sensation of like tingling or sparkling. I love it's peaceful yet energised feeling that feels instilled with me afterwards, and it creates this sense or activation of movement

2. In terms of the actions it inspires in me; it really drives me to connect more deeply with people through conversations and a sense of presence. It inspires me to write and share the energy through the written word and often influences the perspective of which I will write in. And thirdly it also gives me a grounded sense of a 'getting sh*t done' vibe - which I channel more proactively based on the project I am working on
---

A very interesting subject.  I struggle with the notion of Hope as (in my mind) it presents itself as an alternative to Action, so passive in nature.

Is there a particular notion or feeling that works hand in hand with action for you?

Yes, I think it would be - excitement, motivation, curiosity, positivity - as in, I can have it done, I believe in myself.  Which makes me think that I might be thinking too much individualistically? Is hope more of a "belief" in the world - in the sense of - the world won't let things go wrong?

hope is whatever it means for you, and maybe collective hope would sit in a very different landscape, which I invite people to also reflect on. 

To reflect on what you have said, from a personal place, for me hope is like the spark, and that might lead into excitement or motivation to do, explore, but it also brings with it a sense of I am here, now, and that makes me want to/allows me to embody it in a way of being and to inform my actions to connect, such as starting this conversation 😃

Is it always there or it ebbs and flows?

---

I'm sensing that it might be helpful to look at hope through its opposite, hopelessness. When I'm hopeless, I have no motivation to act; I feel that whatever goal it is is unreachable, and therefore a waste of time and energy to pursue. So, I can see hope as that quality of spirit that sees a point in continuing towards what is hoped for, either actively through one's own agency, or passively through waiting for some external deliverance. Hopefully not "Waiting for Godot"! But hope itself is not clinging on to any one particular strategy; it is open to what emerges. That's the positive side of hope, to me. Though passive hope is of questionable value.

---

Wonderful teacher Joanna Macy has a framework called Active Hope, and there's a free course that I have found very very useful. I use it with my climate org and other groups in Berlin. 

https://activehope.training

It makes a clear differentiation between a passive form of hope, and active hope.

---

Coming from a Buddhist informed philosophy, I like the idea of 'abandoning hope of fruition' — they also recommend abandoning fear. 

The idea behind abandoning hope is that you don't need hope for anything in the future in order to be present and act wisely and in line with your best judgement. 

We can acknowledge that death can come at any moment, and we can live a life where our only 'hope' is to live each day and each moment with as much love, kindness, generosity and peace as humanly possible. 

I find this very freeing coming from a Roman Catholic background that emphasised hope and putting our salvation in a future, distant, divine something outside of myself.

**Comment by asimong** [comment]
Perhaps it may also be helpful to distinguish hope from its near neighbours: faith and trust. To me, faith is in something or someone: I believe that the good effect I am looking for will come through a certain route or person or chain of causality, without being able to be certain about it. Contrast this with hope, which to me is more open, as I've suggested above. Trust seems to me to be less bound to an outcome; more generalised. If I trust that things will work out, I may not be ascribing any particular agency to myself, but more likely to someone else. On the other hand, I could say I trust that what I do will bring about a certain outcome; but in that sense it seems very similar to saying I have faith in my agency for this.

So: hope seems to me rather more open in two ways. First, that I can still hope for something that I know may not happen. I can hope that the team I follow will win the match; but they might lose. But there is almost nothing I can do about it, beyond cheering from the stands. Second, and I would say more pertinent to this conversation, is that I see hope as being open to the agency of the outcome being either within me or not.  It's certainly saying that what I hope for is what I would value in some way. But whether it comes about through my efforts or not seems to me not to change the quality of the hope itself; only of the means by which it happens. For sure, if I try to bring something about and fail, I could still hope that someone else will make it happen, or that it will simply happen of its own accord, or by chance. But also, if I have been neglecting my own agency and I express my hope in something, someone else might point out to me that I can in fact do something towards it. It seems so common for people to neglect their own agency. The well-known phrase “be the change that you want to see in the world” is also one way of reminding people of their agency in bringing about what they desire, even if it's not clear just how that will happen — we start with ourselves.

And so I'd like to express support for encouragement; for people encouraging each other. A response to "I wish this would happen, but I don't know how it could ever be" might be, for example, "well, here is what you can do; it might not seem much, it might not appear to be significant, but you know, every little bit helps, and if we all play our part (embodying our own agency) then it is so much more likely to happen. So, don't lose hope! It's true that you can't make it happen all by yourself, but together we have a real hope." 

And, by the way, I hope this is in line with Joanna Macy. ;-)

**Comment by asimong** [comment]
And I would now add, on further reflection, some things really are governed by chance, or at least by no clear causality at all, and for those things, passive hope seems to be the only option. And I see nothing wrong with that. If I hope for something, then I might consider how to respond if it indeed comes to pass.

**Reply by martinprogrammer** [reply]
@asimong 
By saying that "some things really are governed by chance" and "at least by no clear causality at all" - would the answer not be to try and figure out the causality (unless you're saying that you don't believe there's causality in some cases)?  Is passive hope in your example a feeling of desperation that one cannot find either motivation or capacity to continue trying to solve the problem or assume control?  In that case, hope would be self-medicating.

As an alternative to hope - is it not possible to disown and disassociate from  the expectation of a positive outcome and move onto one which is within your control.

I'm ready for the extreme examples that you might throw at me. ;-)

**Reply by asimong** [reply]
In what Dave Snowden calls "complicated" cases, yes, for sure, figure out the causality. But complex situations? It's not that there is no causality, it's just that the causality is beyond normal analysis. The system can even be in "chaos" where arbitrarily small changes in the initial conditions cause arbitrarily large effects after time elapses. I'm not advocating for passive hope as any kind of serious answer. What I am saying is that in complex situations, where the causality is impenetrable, it may be wise to hold on to passive hope during intervals where the effects of what one can do actively are unclear to the point of being potentially counterproductive.

Again, I'm not advocating for passive hope as dissociation, either. But the point is, when the situation is too complex to be able to figure out if one's actions are productive or counterproductive, it is wise to wait that out, possibly with small probes to see what the complex situation may respond to. Or careful observation, until some actionable causality is revealed.

Might come back to extreme examples, but I'm not promising. If you'd like to offer examples, that would also be helpful.
